src/components/Cards/Ghost/post/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Link from 'next/link'
+
+import GhostPost from '../../../../interfaces/Ghost/GhostPost'
+
+import { Container, CardHeadline, CardPublishedAt } from './styles'
+
+interface GhostPostCardProps {
```

Comment:
Leí hace poco que cuando una interfaz va a ser exclusivamente de una clase, es mejor o más claro usar un tipo:
https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c

Qué opina @DrankoLQ ?